teps/0031-tekton-bundles-cli.md

```diff
@@ -90,15 +90,6 @@ tkn bundle push <REF> [BUNDLE_OBJECT] -f [BUNDLE_OBJECT]
 # Fetches a Tekton Bundle and prints its contents in a configurable format. If KIND is specified, will print only
 # objects of the specified kind (eg, Pipeline or Task). If KIND and NAME are specified, will retrieve a specific object.
 tkn bundle get <REF> [KIND] [NAME] --output=[FORMAT]
-
-# The following are amendments to existing commands.
-
-# Retrieves from an image the requested task(s) rather than from the cluster. Will print the result in the specified
-# format, such as yaml or json.
-tkn task list --image=<REF> --output=[FORMAT]
-tkn task get --image=<REF> --output=[FORMAT] [NAME]
-
-# ... the pipeline command receives these changes as well
```

Comment:
Did you decide to not implement these ones?

Reply:
Basically, yes. I figured, the existing 2 commands should be enough for now and I don't recall anyone passionately wanting the extra options in `tkn task` and `tkn pipeline` so I figured I could add those later if they make sense.